script.js

```diff
@@ -638,12 +638,44 @@ const validateEmail =(email)=>{
   }
   return true;
 }
-const form = document.querySelector('form')
+const form = document.querySelector('form');
 form.addEventListener('submit',(event)=>{
  // e.preventDefault();
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much more noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻